dalston/block-extends/extend-cover-block.css

```diff
@@ -144,23 +144,27 @@
 	text-decoration: underline;
 }
 
-.wp-block-cover.is-style-hover-reveal .wp-block-cover__inner-container {
+.wp-block-cover.is-style-hover-reveal .wp-block-cover__inner-container,
+.wp-block-cover.is-style-hover-reveal .wp-block-cover-text {
```

Comment:
The addition of `.wp-block-cover-text` is to fix the reveal in old versions of Gutenberg where it didn't have the InnerBlocks in the Cover yet.